src/components/NewTodo/NewTodo.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './NewTodo.css';
+
+export class NewTodo extends React.Component {
+  state = {
+    users: this.props.users,
+    title: '',
+    userId: 0,
+    id: 3,
+    errorTitle: false,
+    errorUser: false,
+  }
+
+  onSubmit = (e) => {
+    e.preventDefault();
+
+    const { userId, id, title } = this.state;
+
+    if (title === '') {
+      this.setState({
```

Comment:
use setState once for one method

Reply:
write all the code inside setState callback

Reply:
i can't validate properties in setState's callback

Reply:
why? use prevState